libbpf-rs/src/query.rs

```diff
@@ -586,33 +624,37 @@ impl Iterator for BtfInfoIter {
     }
 }
 
+/// Information about a raw tracepoint.
 #[derive(Debug, Clone)]
-// TODO: Document members.
-#[allow(missing_docs)]
 pub struct RawTracepointLinkInfo {
+    /// The name of the raw tracepoint
     pub name: String,
+    /// Length (in bytes) of the raw tracepoint name
+    pub name_len: u32,
```

Comment:
Why is this member needed? We do have the name and it has a length, no? This seems like useless duplication of information to me.